modules/packages/Python.chpl

```diff
@@ -358,6 +358,47 @@ module Python {
   */
   config param checkExceptions = true;
 
+  /*
+    Internal internal object to handle the GIL. This is used to
```

Comment:
Duplicate internals here (internal inception!)

Reply:
I just wanted to be really explicit :smile: